### PR TITLE
Update RHEL 9 STIG to V2R2

### LINKED ISSUE #12547: Update RHEL9 DISA STIG to V2R2
#### Description of problem:
Update /products/rhel9/profiles/stig.profile, /products/rhel9/profiles/stig_gui.profile, and /shared/content/disa_stig-rhel9* to support [Red Hat Enterprise Linux 9 STIG - Ver 2, Rel 2](https://dl.dod.cyber.mil/wp-content/uploads/stigs/zip/U_RHEL_9_V2R2_STIG.zip) (23 October 2024) 

#### SCAP Security Guide Version:
0.1.75 

#### Operating System Version:
RHEL 9.4

### PR DESCRIPTION
**Pull request by Mab879**
#### Description:

* Update RHEL 9 STIG to V2R2

#### Rationale:

* Fixes #12547

### CODE REVIEW

File: controls/stig_rhel9.yml [hunk 1]
```diff
@@ -1847,7 +1847,8 @@ controls:
             - medium
         title: RHEL 9 SSH daemon must be configured to use system-wide crypto policies.
         rules:
-            - configure_ssh_crypto_policy
+            - harden_sshd_ciphers_opensshserver_conf_crypto_policy
```
Comment (vojtapolasek) [inline controls/stig_rhel9.yml:1850]: I am not sure if this rule should be here. Reading the relevant STIG, it seems that the audit part does not mention desired ciphers at all. But the rule you use actually checks for exact ciphers in files provided by crypto-policies package. I think we need a new rule as this STIG checks rather for correct include directives than for actual ciphers configured.

**Review by vojtapolasek — CHANGES_REQUESTED** [review]

File: controls/stig_rhel9.yml (hunk 1 above)
Comment (Mab879) [reply controls/stig_rhel9.yml]: New rule added.

**Review by vojtapolasek — APPROVED** [review]

**Comment by vojtapolasek** [comment]
LGTM now, thank you. Failing tests fail because new rules are currently present only in rhel9 benchmark.

**Comment by christopher-davidson** [comment]
@Mab879 I believe this PR should have also bumped "version: V2R1" to "version: V2R2" in /products/rhel9/profiles/stig.profile/ and /products/rhel9/profiles/stig_gui.profile/

**Comment by Mab879** [comment]
> @Mab879 I believe this PR should have also bumped "version: V2R1" to "version: V2R2" in /products/rhel9/profiles/stig.profile/ and /products/rhel9/profiles/stig_gui.profile/

Thanks for pointing this out. Fixed in #12597.

**Pull request merged**